### PR TITLE
docs(example-soap-calculator): fix wrong links to docs and image path

### LINKED ISSUE #1638: SOAP Tutorial LB4 Doc Page Not Found
When browsing SOAP calculator Web Service integration tutorial (https://loopback.io/doc/en/lb4/soap-calculator-tutorial.html) some tutorial subsections don't work for example:

page found when browsing main page: https://loopback.io/doc/en/lb4/soap-calculator-add-datasource.html
page not found when browsing left menu: https://loopback.io/doc/en/lb4/soap-calculator-tutorial-add-datasource.html

Also when browsing main tutorial page, last two steps are not available either:
https://loopback.io/doc/en/lb4/soap-calculator-tutorial-make-service-available.md
https://loopback.io/doc/en/lb4/soap-calculator-run-and-and-test.md

_See [Reporting Issues](http://loopback.io/doc/en/contrib/Reporting-issues.html) for more tips on writing good issues_

### PR DESCRIPTION
**Pull request by marioestradarosa**
fix #1638

## Checklist

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

File: docs/site/soap-calculator-tutorial-add-controller.md [hunk 1]
```diff
@@ -3,12 +3,12 @@ lang: en
...
 keywords: LoopBack 4.0, LoopBack 4
 sidebar: lb4_sidebar
-permalink: /doc/en/lb4/soap-calculator-add-controller.html
+permalink: /doc/en/lb4/soap-calculator-tutorial-add-controller.html
 ---
 
 ### Add a Controller
 
-![soap-calculator-add-service](../../docs/img/loopback-example-soap-calculator_figure4.png)
+![soap-calculator-add-service](../img/loopback-example-soap-calculator_figure4.png)
```
Comment (virkt25) [inline docs/site/soap-calculator-tutorial-add-controller.md]: The images right now live at `loopback-next/docs/img` ... this folder is not copied into `loopback.io` during the jekyll build process. The images in the above directory need to be moved to `/loopback-next/docs/site/imgs/`. 

You can test this locally by doing the following from `loopback-next` top level. 
```
npm run build:site
cd sandbox/loopback.io
npm start
// Visit http://127.0.0.1:4001/doc/en/lb4/soap-calculator-tutorial.html#overview
```

Then this should be good to land with the links fixed. Thanks @marioestradarosa !

Comment (marioestradarosa) [reply docs/site/soap-calculator-tutorial-add-controller.md]: Thanks for the explanation. 👍

**Comment by virkt25** [comment]
Thanks for the contribution once again @marioestradarosa ❤️ 🎉

**Comment by marioestradarosa** [comment]
Thanks @virkt25 , now I know how to test the doc site 👍.

**Pull request merged**